src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -4171,6 +4171,69 @@ function Test-ApplicationGatewayFirewallPolicyManagedRuleGroupOverrideEmptyRule
 	}
 }
 
+<#
+.SYNOPSIS
+Application gateway v2 top level waf tests in ManagedRuleGroupOverride with sensitivity
+#>
+function Test-ApplicationGatewayFirewallPolicyManagedRuleGroupOverrideWithSensitivity
```

Comment:
Please add a test trying to retrieve the default override config for CRS/BOT , the sensitivity should not be populated(not even "None")

Reply:
tested manually